### ISSUE TITLE
Attached device does not appear in the Local Devices section

### ISSUE THREAD

**Issue by wu2s** [aredn/aredn#1679]
### Describe the bug
I have a Raspberry Pi web server connected via wired LAN to a Mikrotik hAP AC3. The Pi is connected to port 3 on the hAP. The front panel on the hAP shows port 3 lit up. The Pi has obtained a LAN IP from the hAP as 10.156.233.74. The Ports & Xlinkls section on the hAP shows port 3 active (green background instead of grey).

Even though successfully connected and addressed, there are no devices shown in the Local Devices section on the hAP>

- [ ] Does this issue still occur on the latest version? (**Required**)
- [ ] Yes - the hAP is running NB 20241107

### Screenshots
Please include a screenshot of your system information (if possible) if the specific system environment is relevant to the bug.

### Additional context
![Screenshot 2024-11-07 at 21 15 46](https://github.com/user-attachments/assets/7394786e-a797-4eb4-8c0c-d20b3155d66d)
![Screenshot 2024-11-07 at 21 16 26](https://github.com/user-attachments/assets/0c77d4d0-3e15-49b6-8e3f-a46b4fd79ab2)

**Comment by K5RA** [comment]
Do you have to have DHCP reservations and active leases for your Local Devices?
My hAP ac lite shows my RP5B and a VoIP phone, both of which have DHCP entries.

**Comment by aanon4** [comment]
Could you re-add the screenshot but dont cut off the LAN DHCP information?

**Comment by aanon4** [comment]
Also, please add the support data. Thanks.

**Comment by wu2s** [comment]
Yes the hAP shows 1 active lease.
Screenshot and support data are attached.
[supportdata-WU2S-H3-X-2024-11-8-7-51.tar.zip](https://github.com/user-attachments/files/17679620/supportdata-WU2S-H3-X-2024-11-8-7-51.tar.zip)
![Screenshot 2024-11-08 at 07 50 54](https://github.com/user-attachments/assets/6ddd1ce7-b2e4-44db-9b3a-8bfd13944a0c)

**Comment by K5RA** [comment]
If you make the DHCP Address Reservation with device name and MAC address, and DoNotPropagate unchecked, that device will appear as a Local Device.  Active lease apparently not required.

**Comment by wu2s** [comment]
@K5RA Thank you for the guidance. I determined the MAC address of the Raspberry PI and then went to the hAP to reserve the address. At first, the Pi did not show up in the Local Devices list, even after wait several minutes and doing a screen refresh. Then I went back to the LAN DHCP address reservations and UNCHECKED the Do Not Propagate box as you advised, and the Raspberry Pi instantly appeared in the Local Devices list.
